<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1120</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qt Chat Client</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <layout class="QVBoxLayout" name="outerLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QFrame" name="windowCard">
      <property name="styleSheet">
       <string notr="true">QFrame#windowCard{background:#FFFFFF;border:1px solid #F0F0F0;border-radius:8px;}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <layout class="QVBoxLayout" name="pageLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="titleBar">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>36</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>36</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame#titleBar{background:#FAFAFA;border:none;border-bottom:1px solid #F2F2F2;} QToolButton{border:none;background:transparent;color:#777777;min-width:34px;min-height:26px;} QToolButton:hover{background:#F0F0F0;} QToolButton#closeButton:hover{background:#E81123;color:white;} QLabel{color:#A0A0A0;}</string>
         </property>
         <layout class="QHBoxLayout" name="titleLayout">
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="topMargin">
           <number>4</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>4</number>
          </property>
          <item>
           <widget class="QLabel" name="titleTextLabel">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="titleSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="minButton">
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="maxButton">
            <property name="text">
             <string>□</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="closeButton">
            <property name="text">
             <string>×</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="rootLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="leftContainer">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="sessionContainer">
           <property name="minimumSize">
            <size>
             <width>280</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="chatContainer"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
